Edit main landing pages for style and clarity (#3848)
open-telemetry/opentelemetry.io: content/en/docs/what-is-opentelemetry.md

---
title: What is OpenTelemetry?
description: A short explanation of what OpenTelemetry is and isn't.
aliases: [/about, /docs/concepts/what-is-opentelemetry, /otel]
weight: -1
---

OpenTelemetry is an
[Observability](/docs/concepts/observability-primer/#what-is-observability)
framework and toolkit designed to create and manage telemetry data such as
[traces](/docs/concepts/signals/traces/),
[metrics](/docs/concepts/signals/metrics/), and
[logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
tool-agnostic, meaning that it can be used with a broad variety of Observability
back ends, including open source tools like
[Jaeger](https://www.jaegertracing.io/) and
[Prometheus](https://prometheus.io/), as well as commercial offerings.

OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
commercial vendors. OpenTelemetry is focused on the generation, collection,
management, and export of telemetry data. The storage and visualization of that
data is intentionally left to other tools.

## What is observability?

> Observability is the ability to understand the internal state of a system by

Reviewer: ```suggestion
[Observability](/docs/concepts/observability-primer/#what-is-observability) is the ability to understand the internal state of a system by
```

[logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
tool-agnostic, meaning that it can be used with a broad variety of Observability
back ends, including open source tools like
[Jaeger](https://www.jaegertracing.io/) and
[Prometheus](https://prometheus.io/), as well as commercial offerings.

OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
commercial vendors. OpenTelemetry is focused on the generation, collection,
management, and export of telemetry data. The storage and visualization of that
data is intentionally left to other tools.

## What is observability?

[Observability](/docs/concepts/observability-primer/#what-is-observability) is
the ability to understand the internal state of a system by examining its
outputs. In the context of software, this means being able to understand the
internal state of a system by examining its telemetry data, which includes
traces, metrics, and logs.

To make a system observable, it must be instrumented. That is, the code must
emit [traces](/docs/concepts/observability-primer/#distributed-traces),
[metrics](/docs/concepts/observability-primer/#reliability--metrics), and

> [logs](/docs/concepts/observability-primer/#logs). The instrumented data must

Reviewer: I wonder if we should link to the signal pages and not to the chapters within the observability primer? Also we could put a link to https://opentelemetry.io/docs/concepts/instrumentation/ when the word instrumented is mentioned first.

---
title: What is OpenTelemetry?
description: A short explanation of what OpenTelemetry is and isn't.
aliases: [/about, /docs/concepts/what-is-opentelemetry, /otel]
weight: -1
---

OpenTelemetry is an
[Observability](/docs/concepts/observability-primer/#what-is-observability)
framework and toolkit designed to create and manage telemetry data such as
[traces](/docs/concepts/signals/traces/),
[metrics](/docs/concepts/signals/metrics/), and
[logs](/docs/concepts/signals/logs/). Crucially, OpenTelemetry is vendor- and
tool-agnostic, meaning that it can be used with a broad variety of Observability

> back ends, including open source tools like

Reviewer 2: ```suggestion
backends, including open source tools like
```

For a justification, see:

- #3882

Author: Agreed in referenced PR!

[Jaeger](https://www.jaegertracing.io/) and
[Prometheus](https://prometheus.io/), as well as commercial offerings.

> OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
> commercial vendors. OpenTelemetry is focused on the generation, collection,
> management, and export of telemetry data. The storage and visualization of that
> data is intentionally left to other tools.

Reviewer 3: ```suggestion
OpenTelemetry is not an observability back end like Jaeger, Prometheus, or other
commercial vendors. OpenTelemetry is focused on the generation, collection,
management, and export of telemetry. A major goal of OpenTelemetry is that you
can easily instrument your applications or systems, no matter their language, infrastructure,
or runtime environment. Crucially, the storage and visualization of telemetry is intentionally
left to other tools.
```

Author: Good point, I always meant to ask about "telemetry data" - we use this expression elsewhere and I feel it's redundant.

## What is observability?

[Observability](/docs/concepts/observability-primer/#what-is-observability) is
the ability to understand the internal state of a system by examining its
outputs. In the context of software, this means being able to understand the
internal state of a system by examining its telemetry data, which includes
traces, metrics, and logs.

To make a system observable, it must be
[instrumented](/docs/concepts/instrumentation). That is, the code must emit
[traces](/docs/concepts/signals/traces/),

> [metrics](/docs/concepts/signals/metrics/), and

Reviewer 3: ```suggestion
[metrics](/docs/concepts/signals/metrics/), and/or
```

Author: Went with just "or" here following Google's recommendation: https://developers.google.com/style/slashes#and-or
